packages/shared/src/components/onboarding/OnboardingPlusStep.tsx

```diff
@@ -53,12 +53,12 @@ export const OnboardingPlusStep = ({
   const { productOptions } = usePaymentContext();
   const item = useMemo(
     () =>
-      productOptions
-        .toSorted(({ appsId, duration }) => {
+      [...productOptions]
+        .sort(({ appsId, duration }) => {
           if (appsId === PlusPriceTypeAppsId.EarlyAdopter) {
             return -1;
           }
-          return duration === PlusPriceType.Yearly ? 0 : 1;
+          return duration === PlusPriceType.Yearly ? -1 : 0;
```

Comment:
why was this reversed?

Reply:
I've tested and monthly need to go after yearly, not it works as intended ✔️